### PR TITLE
Feature/bank integration

### PR DESCRIPTION
**Pull request by samjeckert**
Take a look (top of README). I did some research on ACH payments. Looks like we need to be a little more legit before we can set that up. Want to merge this just to get an initial commit on the books, will be creating a wrapper for Plaid next (will try to see how far I can get on demo mode with Stripe as well).

TBH most of this commit is starter code, will be removed/modified soon.

### CODE REVIEW

**Comment by piercefreeman** [comment]
FWIW, I got an invite to Stripe's Atlas, which provides a very affordable LLC creation and most of the legal docs we'd need to set up these company accounts.

File: third-party/bank-integration/templates/index.ejs [hunk 1]
```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Plaid Walkthrough Example</title>
+    <link rel="stylesheet" type="text/css" href="static/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div id="banner">
```
Comment (piercefreeman) [inline third-party/bank-integration/templates/index.ejs]: We'll probably want to integrate this with our React stack

**Comment by samjeckert** [comment]
That sounds great. We have the funds, might as well go ahead on that. If we do that (and throw up a quick startupy website and create a solvence bank account) we can set this up and ill be happy AF (Ex: **Figure 1**).

Figure 1: image

File: third-party/bank-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Stripe + Plaid Bank Integration
+
+This repository defines a pythonic wrapper around the functionality provided by Plaid and Stripe. Namely we can use this
+class in order to add Fiat currency to our exchange.
+
+We use Stripe to facilitate ACH payments (electronic payments) (note that this must be enabled),
+and then we use Plaid in order to verify/query bank accounts seamlessly without microtransactions
+and the like. Together they allow us to add Fiat to Solvence.
```
Comment (piercefreeman) [inline third-party/bank-integration/README.md]: So tldr; stripe would handle the actual debiting of funds, but Plaid would be used to present the frontend to authenticate with their account?

File: third-party/bank-integration/templates/index.ejs (hunk 1 above)
Comment (samjeckert) [reply third-party/bank-integration/templates/index.ejs]: Gotcha. Probably going to just create a Node/Python wrapper. Will talk with geoff

File: third-party/bank-integration/README.md (hunk 2 above)
Comment (samjeckert) [reply third-party/bank-integration/README.md]: Yes. Stripe should handle the actual payments, and plaid provides an interface for authentication with customer bank accounts.

**Pull request merged**